robottelo/cli/base.py

```diff
@@ -194,7 +194,8 @@ def execute(cls, command, user=None, password=None, output_format=None,
         """Executes the cli ``command`` on the server via ssh"""
         user, password = cls._get_username_password(user, password)
 
-        cmd = u'LANG={0} hammer -v -u {1} -p {2} {3} {4}'.format(
+        # add time to measure hammer performance
+        cmd = u'LANG={0} time -p hammer -v -u {1} -p {2} {3} {4}'.format(
```

Comment:
The time command should be added if a configuration is set. By adding it here it will add contents to stderr and will make many tests fail. With that said, maybe we need a bigger refactor on Robottelo's CLI.